.github/workflows/pimcore-demo.yml

```diff
@@ -20,10 +20,10 @@ jobs:
     strategy:
         matrix:
             include:
-                - { php-version: 8.2, database: "mariadb:10.11", pimcore_version: "", experimental: true }
-                - { php-version: 8.2, database: "mariadb:10.11", pimcore_version: "", pimcore_upgrade_version: "11.x-dev as 11.0.0", experimental: true }
-                - { php-version: 8.2, database: "mysql:latest", pimcore_version: "", experimental: true }
-                - { php-version: 8.2, database: "mysql:latest", pimcore_version: "", pimcore_upgrade_version: "11.x-dev as 11.0.0", experimental: true }
+                - { php-version: 8.3, database: "mariadb:10.11", pimcore_version: "", experimental: true }
+                - { php-version: 8.3, database: "mariadb:10.11", pimcore_version: "", pimcore_upgrade_version: "11.x-dev as 11.0.0", experimental: true }
+                - { php-version: 8.3, database: "mysql:latest", pimcore_version: "", experimental: true }
+                - { php-version: 8.3, database: "mysql:latest", pimcore_version: "", pimcore_upgrade_version: "11.x-dev as 11.0.0", experimental: true }
```

Comment:
can i revert these changes? it seems that in the https://github.com/pimcore/demo/pull/568/files#diff-d7b672fd527bb6f82ceb3c23b6f417911c595d020f4a14aac07cad73936dbf0cR38 we want to test both 8.2 and 8.3, or is it fine to just keep the latest?